docs/tutorial/quickstart.md

```diff
@@ -0,0 +1,377 @@
+}
+artifactType := "application/vnd.example+type"
+desc, err := oras.PackManifest(ctx, repo, oras.PackManifestVersion1_1, artifactType, packOpts)
+if err != nil {
+	panic(err)
+}
+tag := "quickstart"
+err = repo.Tag(ctx, desc, tag)
+if err != nil {
+	panic(err)
+}
+fmt.Println("Pushed and tagged manifest")
+```
+
+## Fetch the manifest from the repository by tag
+
+The following code snippet demonstrates how to fetch a manifest from the repository by its tag. First, the descriptor associated with the tag is resolved using [(*Repository) Resolve](https://pkg.go.dev/oras.land/oras-go/v2@v2.5.0/registry/remote#Repository.Resolve). Then, the content is fetched using [(*Repository) Fetch](https://pkg.go.dev/oras.land/oras-go/v2@v2.5.0/registry/remote#Repository.Fetch). Finally, the content is read using [ReadAll](https://pkg.go.dev/oras.land/oras-go/v2@v2.5.0/content#ReadAll) from the [content](https://pkg.go.dev/oras.land/oras-go/v2@v2.5.0/content) package. Paste the code into `main.go` after the last section.
+
+```
+// 5. Fetch the manifest from the repository by tag
+desc, err = repo.Resolve(ctx, tag)
+if err != nil {
+	panic(err)
+}
+rc, err := repo.Fetch(ctx, desc)
+if err != nil {
+	panic(err)
+}
+defer rc.Close() // don't forget to close
+fetchedManifestContent, err := content.ReadAll(rc, desc)
+if err != nil {
+	panic(err)
+}
+fmt.Println(string(fetchedManifestContent))
+```
+
+## Parse the fetched manifest content and get the layers
+The following code snippet demonstrates how to parse the fetched manifest content and get the layers. Paste the code into `main.go` after the last section.
+
+```
+// 6. Parse the fetched manifest content and get the layers
+var manifest v1.Manifest
+if err := json.Unmarshal(fetchedManifestContent, &manifest); err != nil {
+	panic(err)
+}
+layers := manifest.Layers
+for _, layer := range layers {
+	rc, err = repo.Fetch(ctx, layer)
+	if err != nil {
+		panic(err)
+	}
+	defer rc.Close() // don't forget to close
+	layerContent, err := content.ReadAll(rc, layer)
+	if err != nil {
+		panic(err)
+	}
+	fmt.Println(string(layerContent))
+}
+```
+
+## Copy the artifact to local OCI layout directory from the repository
+
+The following code snippet demonstrates how to copy an artifact from the repository by its tag and save it to the current directory in the [OCI layout](https://github.com/opencontainers/image-spec/blob/main/image-layout.md) format. The copy operation is performed using the [Copy](https://pkg.go.dev/oras.land/oras-go/v2#Copy) function. Paste the code into `main.go` after the last section.
+
+```
+// 7. Pull the manifest to local OCI layout directory with a tag "quickstartOCI"
+ociDir, err := os.MkdirTemp(".", "oras_oci_example_*")
+if err != nil {
+	panic(err)
+}
+ociTarget, err := oci.New(ociDir)
+if err != nil {
+	panic(err)
+}
+desc, err = oras.Copy(ctx, repo, tag, ociTarget, "quickstartOCI", oras.DefaultCopyOptions)
+if err != nil {
+	panic(err)
+}
+fmt.Println(desc.Digest)
```

Comment:
nit: Should we print a bit more message like "Copied <tag>, digest: <digest>"?

Reply:
The digest changes and `PackManifest` gives a different value for the "org.opencontainers.image.created" each time. It's better to not print the digest as it may be confusing to users that the digest is different from theirs.